examples/kitti/detectnet_network.prototxt

```diff
@@ -0,0 +1,2538 @@
+# DetectNet network
+
+# Data/Input layers
+name: "DetectNet"
+layer {
+  name: "train_raw_data"
+  type: "Data"
+  top: "raw_data"
+  data_param {
+    backend: LMDB
+    source: "examples/kitti/kitti_train_images.lmdb"
+    batch_size: 10
+  }
+  include: { phase: TRAIN }
+}
+layer {
+  name: "train_raw_label"
+  type: "Data"
+  top: "raw_label"
+  data_param {
+    backend: LMDB
+    source: "examples/kitti/kitti_train_labels.lmdb"
+    batch_size: 10
+  }
+  include: { phase: TRAIN }
+}
+layer {
+  name: "val_raw_data"
+  type: "Data"
+  top: "raw_data"
+  data_param {
+    backend: LMDB
+    source: "examples/kitti/kitti_test_images.lmdb"
+    batch_size: 6
+  }
+  include: { phase: TEST stage: "val" }
+}
+layer {
+  name: "val_raw_label"
+  type: "Data"
+  top: "raw_label"
+  data_param {
+    backend: LMDB
+    source: "examples/kitti/kitti_test_labels.lmdb"
+    batch_size: 6
+  }
+  include: { phase: TEST stage: "val" }
+}
+layer {
+  name: "deploy_data"
+  type: "Input"
+  top: "data"
+  input_param {
+    shape {
+      dim: 1
+      dim: 3
+      dim: 384
+      dim: 1248
+    }
+  }
+  include: { phase: TEST not_stage: "val" }
+}
+
+# Data transformation layers
+layer {
+  name: "train_transform"
+  type: "DetectNetTransformation"
+  bottom: "raw_data"
+  bottom: "raw_label"
+  top: "data"
+  top: "label"
+  detectnet_groundtruth_param: {
+    stride: 16
+    scale_cvg: 0.4
+    gridbox_type: GRIDBOX_MIN
+    coverage_type: RECTANGULAR
+    max_cvg_len: 50
+    min_cvg_len: 16
+    obj_norm: true
+    image_size_x: 1248
+    image_size_y: 384
+    crop_bboxes: true
+  }
+   detectnet_augmentation_param: {
+    crop_prob: 1
+    shift_x: 32
+    shift_y: 32
+    flip_prob: 0.5
+    rotation_prob: 0
+    max_rotate_degree: 5
+    scale_prob: 0.33
+    scale_min: 0.7
+    scale_max: 1.3
+    hue_rotation_prob: 0.9
+    hue_rotation: 45
+    desaturation_prob: 0.9
+    desaturation_max: 0.8
+  }
+  transform_param: {
+    mean_value: 127
+  }
+  include: { phase: TRAIN }
+}
+layer {
+  name: "val_transform"
+  type: "DetectNetTransformation"
+  bottom: "raw_data"
+  bottom: "raw_label"
+  top: "data"
+  top: "label"
+  detectnet_groundtruth_param: {
+    stride: 16
+    scale_cvg: 0.4
+    gridbox_type: GRIDBOX_MIN
+    coverage_type: RECTANGULAR
+    max_cvg_len: 50
+    min_cvg_len: 16
+    obj_norm: true
+    image_size_x: 1248
+    image_size_y: 384
+    crop_bboxes: false
+  }
```

Comment:
Please change this to 
    stride: 16
    scale_cvg: 0.4
    gridbox_type: GRIDBOX_MIN
    coverage_type: RECTANGULAR
    min_cvg_len: 20
    obj_norm: true
    image_size_x: 1248
    image_size_y: 384
    crop_bboxes: false